reo/validators.py

```diff
@@ -1255,6 +1259,26 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                     # Use 2017 b/c it is most recent year that starts on a Sunday and all reference profiles start on
                     # Sunday
 
+                if real_values.get('outage_utility_name') != None and real_values.get('use_default_outage') == False:
+                    self.general_warnings.append("Overriding use_default_outage value, using the default outage duration")
+                    self.update_attribute_value(object_name_path, number, 'use_default_outage', True)
+                    real_values['use_default_outage'] = True
+                
+                if real_values.get('use_default_outage') == True:
+                    reliability = pd.read_csv('reo/src/data/Reliability_2019.csv', header = 1)
+                    utility_names = reliability['Utility Name']
+                    print(utility_names)
+                    if real_values.get('outage_utility_name') not in utility_names:
+                        self.general_warnings.append("Invalid outage_utility_name, cannot use default outage")
+                        self.update_attribute_value(object_name_path, number, 'use_default_outage', False)
+                        real_values['use_default_outage'] = False
+
+                if real_values.get('use_default_outage') == True and (real_values.get('outage_start_time_step') != None and real_values.get('outage_end_time_step' != None)):
```

Comment:
Based on what happens in LoadProfile.py right now, a user can specify `use_default_outage` as `true` and pass in `None` for `outage_start_time_step` and `outage_end_time_step`. This will cause an error when you try to subtract the duration from a `None` value. You might want to catch this case here and raise an error message to the user, or otherwise handle it.